pkg/operator/staticpod/installerpod/copy_test.go

```diff
@@ -1,135 +0,0 @@
-package installerpod
-
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
-
-	"k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/kubernetes/fake"
-	ktesting "k8s.io/client-go/testing"
-)
-
-func addFakeSecret(name string) *v1.Secret {
-	secret := &v1.Secret{}
-	secret.Name = name
-	secret.Namespace = "test"
-	return secret
-}
-
-func TestGetSecretWithRetry(t *testing.T) {
```

Comment:
removing this instead of moving it looks suspicious.

Reply:
i promise to add unit tests to the new helpers :-)

Reply:
(also the retry is heavily tested already)